### PR TITLE
(MAINT) Cherry-pick for Everett

### PR DESCRIPTION
**Pull request by senior**
These are all of the non-stockpile related commits from master that are going out in 4.2.4 release that we will cut off of master

### CODE REVIEW

**Comment by senior** [comment]
@rbrw @wkalt ping

**Comment by senior** [comment]
@camlow325 I think you said you wanted to be pinged on this to double check the EZBake update. PING

File: resources/ext/config/logback.xml [hunk 1]
```diff
@@ -7,10 +7,13 @@
...
     <appender name="F1" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>/var/log/puppetlabs/puppetdb/puppetdb.log</file>
-
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-            <fileNamePattern>/var/log/puppetlabs/puppetdb/puppetdb-%d{yyyy-MM-dd}.log.gz</fileNamePattern>
+        <append>true</append>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>/var/log/puppetlabs/puppetdb/puppetdb-%d{yyyy-MM-dd}.%i.log.gz</fileNamePattern>
+            <!-- each file should be at most 10MB, keep 90 days worth of history, but at most 1GB total-->
+            <maxFileSize>10MB</maxFileSize>
```
Comment (camlow325) [inline resources/ext/config/logback.xml]: I think we agreed that we wanted to use `200MB` as the `maxFileSize` to minimize the number of possible rotations within a day somewhat.  That's what we did across the other components.  @wkalt - does that sound right to you?

File: resources/ext/config/logback.xml [hunk 2]
```diff
@@ -27,6 +30,28 @@
...
         level="info"/>
 
+    <appender name="STATUS" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>/var/log/puppetlabs/puppetdb/puppetdb-status.log</file>
+        <append>true</append>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <!-- rollover daily -->
+            <fileNamePattern>/var/log/puppetlabs/puppetdb/puppetdb-status-%d{yyyy-MM-dd}.%i.log.gz</fileNamePattern>
+            <!-- each file should be at most 10MB, keep 90 days worth of history, but at most 1GB total-->
+            <maxFileSize>10MB</maxFileSize>
```
Comment (camlow325) [inline resources/ext/config/logback.xml]: Same comment here about `200MB` rather than `10MB`.

File: resources/ext/config/request-logging.xml [hunk 3]
```diff
@@ -1,6 +1,14 @@
 <configuration debug="false">
-    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+    <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>/var/log/puppetlabs/puppetdb/puppetdb-access.log</file>
+        <append>true</append>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>/var/log/puppetlabs/puppetdb/puppetdb-access-%d{yyyy-MM-dd}.%i.log.gz</fileNamePattern>
+            <!-- each file should be at most 10MB, keep 90 days worth of history, but at most 1GB total-->
+            <maxFileSize>10MB</maxFileSize>
```
Comment (camlow325) [inline resources/ext/config/request-logging.xml]: Same comment here about `200MB` rather than `10MB`.

**Pull request merged**